Build/build.ps1

```diff
@@ -166,7 +169,10 @@ function Update-CodeCoveragePercentInTextFile {
         [int] $CodeCoveragePercent = 0,
 
         [Parameter(Mandatory = $true)]
-        [string] $TextFilePath
+        [string] $TextFilePath,
```

Comment:
Where is this parameter used in the function ?

Reply:
$TextFilePath is used for getting and setting the contents of it on lines 189-191
```
    $readMeContent = Get-Content $TextFilePath
    $readMeContent = $readMeContent -replace "\*\*$ModuleName\*\* !\[Coverage\].+\)", "**$ModuleName** ![Coverage](https://img.shields.io/badge/coverage-$CodeCoveragePercent%25-$badgeColor.svg?maxAge=60)"
    $readMeContent | Set-Content -Path $TextFilePath
```